src/systemverilog/systemverilog.ts

```diff
@@ -393,10 +391,16 @@ export const language = <ILanguage>{
 		],
 
 		string: [
-			[/[^\\"]+/, 'string'],
-			[/@escapes/, 'string.escape'],
-			[/\\./, 'string.escape.invalid'],
-			[/"/, 'string', '@pop']
+			[/"/, 'string.escape', '@dblStringBody']
+		],
+		dblStringBody: [
```

Comment:
What does dblString stand for?

Reply:
its just dbl for double quote strings. I just kept the name, since I used the syntax for a language with single and double quote strings as reference.